lib/mix/lib/mix/tasks/help.ex

```diff
@@ -169,4 +186,20 @@ defmodule Mix.Tasks.Help do
       end
     end)
   end
+
+  defp build_alias_doc_list(aliases) do
+    Enum.reduce(aliases, {[], 0}, fn {alias_name, task_name}, {docs, max} ->
+      doc = "Alias for " <> task_str(task_name)
+      task = "mix " <> alias_name
+      {[{task, doc} | docs], max(byte_size(task), max)}
+    end)
+  end
+
+  defp task_str(task_name) when is_list(task_name) do
+    task_strs = Enum.map(task_name, &task_str/1) |> Enum.join(", ")
+    "[" <> task_strs <> "]"
```

Comment:
I wonder about the format here with the list, but I'm not really sure what would be better.

Reply:
I would say we should just call `inspect`. It is the simplest implementation and it would be similar to what we have today. But I am not sure.

I guess the formatting was the biggest concern for including aliases in the `mix help`, as they can get quite verbose. Another option is to just have a simple text: "# Alias defined in mix.exs" and call it a day.

Thoughts?